src/dialogs/more-info/more-info-related-info.ts

```diff
@@ -0,0 +1,133 @@
+import { HassEntity } from "home-assistant-js-websocket";
+import { css, html, LitElement, TemplateResult } from "lit";
+import { customElement, property } from "lit/decorators";
+import { styleMap } from "lit/directives/style-map";
+import memoizeOne from "memoize-one";
+import { fireEvent } from "../../common/dom/fire_event";
+import { computeStateDisplay } from "../../common/entity/compute_state_display";
+import { stateColor } from "../../common/entity/state_color";
+import { stateIconPath } from "../../common/entity/state_icon_path";
+import "../../components/ha-chip";
+import "../../components/ha-chip-set";
+import "../../components/ha-icon";
+import { HomeAssistant } from "../../types";
+
+declare global {
+  interface HASSDomEvents {
+    "related-entity": { entityId: string };
+  }
+}
+
+@customElement("more-info-related-info")
+class MoreInfoContent extends LitElement {
+  @property({ attribute: false }) public hass?: HomeAssistant;
+
+  @property({ attribute: false }) public stateObj?: HassEntity;
+
+  private _relatedEntities = memoizeOne((entityId: string) => {
```

Comment:
This is not related entities, but device entities.

Related entities could return more

Reply:
(I think device entities is great btw, we should just rename)